common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3048,6 +3048,12 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_RECURSIVE_CTE" : {
```

Comment:
nit: error class names must be sorted in this file, so we should put it before `INVALID_REGEXP_REPLACE`